language/move-vm/runtime/Cargo.toml

```diff
@@ -12,31 +12,32 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-better_any = "0.1.1"
-fail = "0.4.0"
-once_cell = "1.7.2"
-parking_lot = "0.11.1"
-sha3 = "0.9.1"
-tracing = "0.1.26"
+better_any = { version = "0.1", default-features = false }
```

Comment:
There is a 0.2 version of the crate - could we update it?

Reply:
not in this case - I should have left a comment about it. v0.2 breaks `cargo test`.
I didn't want to fix it - since the error is the same in the move lang repo.